stylesheets/components/lists/lists.css

```diff
@@ -0,0 +1,23 @@
+@import "/main.css";
+
+.spaced-lists{
+    display: flex;
+    width: 100%;
+    justify-content: space-around;
+    gap: 1rem   
```

Comment:
keep flex properties all together, its easy to read when you have multiple properties

Reply:
Thanks for pointing it out, Pankaj! I missed formatting this.